compiler/modules/front.syntax/tests/testParser.bal

```diff
@@ -49,8 +49,8 @@ type SingleStringTokenizerTestCase [string, string];
     dataProvider: sourceFragments
 }
 function testTokenizer(string k, string[] lines) returns error? {
-    SourceFile file = new({ filename: k });
-    Tokenizer tok = new (lines, file);
+    SourceFile file = new({ filename: k }, scanLines(lines));
```

Comment:
Can't we use createSourceFile here?

Reply:
Ah, I mistakenly though it can't be used here. Fixed.